### DISCUSSION TITLE
Set-PnPfolderPermission dependent on language?

### DISCUSSION THREAD

**Discussion by verdura** [pnp/powershell#399] [General]
I have a Powershell script that has this strange behaviour: When I run it it works fine, but when another colleague of mine does it, it gives an error. The command is this:

`Set-PnPfolderPermission -List "Library" -identity "/sites/xxxx/Library/Folder" -User "user@acme.com" -AddRole "Edit"
`The error is:

```
Set-PnPfolderPermission : Cannot locate the level of permission.
At line:226 char:1
+ Set-PnPfolderPermission -List "Library" -identity "/si ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Set-PnPFolderPermission], ServerException
    + FullyQualifiedErrorId : EXCEPTION,PnP.PowerShell.Commands.Files.SetFolderPermission
```
and if I change the `-AddRole` switch to `-AddRole "Editar"` (this is Edit in Portuguese!) it works for him.

I would like to put it to work for both of us. What should I do?

Thanks a lot for your input!

**Comment by mzke** [comment]
yes, the function name is localized. But I have a problem with that. The role list with the get-pnproledefinition command reports a role called "Colaboração" but when I try to use it it returns a role not found message. The same does not happen with the other translated roles names.

```
Name            RoleTypeKind  Hidden Order
----            ------------  ------ -----
Controle Total  Administrator False      1
Design          WebDesigner   False     32
Editar          Editor        False     48
Colaboração     Contributor   False     64
Leitura         Reader        False    128
Acesso Limitado Guest         True     160
```
`Set-PnPListPermission -Identity "$LibraryName" -User $AdGroupID -AddRole "Editar"`
Work
`Set-PnPListPermission -Identity "$LibraryName" -User $AdGroupID -AddRole "Colaboração"`
Don't Work.
